alphagov/govwifi-admin
# Feature/superadmin dashboard

# Description
This introduces a new, empty dashboard that is accessible to super-admins, separate from the current super-admin panel. The admins are now understood as:

* "classic" super-admins: belong to the super-admin organisation;
* new admins: have the `is_super_admin` boolean toggled in their attributes.

This allows us to progressively port the features from the current super-admin views, and ultimately delete our super-admin organisation as it is *not* an organisation in the scope of GovWifi.

# Testing
Half of the diff's additions are new tests but everyone is encouraged to give it a test drive.

1. Clone the branch;
2. Log in as a normal user, mess around;
3. Log in as a classic admin, mess around;
4. Log in as a new admin,  mess around.

See `db/seeds.rb` for account details.
